packages/slice-machine/lib/storybook.ts

```diff
@@ -1,7 +1,7 @@
 import path from "path";
 import TemplateEngine from "ejs";
 import { getOrElseW } from "fp-ts/Either";
-import Files from "./utils/files";
+import Files from "@slicemachine/core/build/node-utils/files";
```

Comment:
nice one! I think everytime we can, we should use the files helper from the core and no longer the one from utils, it's unnecessary duplication :)

Reply:
There's a duplication ticket on the tech debt board, Files isn't event the most common one 😅